logic/scripts/update_token_insight.py

```diff
@@ -15,12 +16,23 @@
 from util import sendgrid_wrapper as sgw
 from util import time_
 
+from redis import Redis
+
+redis_client = Redis.from_url(os.environ['REDIS_URL'])
+
 # NOTE: remember to use lowercase addresses for everything
 
 # token contract addresses
 ogn_contract = "0x8207c1ffc5b6804f6024322ccf34f29c3541ae26"
 dai_contract = "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359"
 
+# ogn wallet addresses
+foundation_reserve_address = "0xe011fa2a6df98c69383457d87a056ed0103aa352"
+team_dist_address = "0xcaa5ef7abc36d5e5a3e4d7930dcff3226617a167"
+investor_dist_address = "0x3da5045699802ea1fcc60130dedea67139c5b8c0"
+dist_staging_address = "0x1a34e5b97d684b124e32bd3b7dc82736c216976b"
+partnerships_address = "0xbc0722eb6e8ba0217aeea5694fe4f214d2e53017"
+ecosystem_growth_address = "0x2d00c3c132a0567bbbb45ffcfd8c6543e08ff626"
```

Comment:
There's one more wallet we probably want to exempt from circulating supply. It's not blocking though.

Reply:
@joshfraser Let me know the address, will add that to this list